pyproject.toml

```diff
@@ -5,10 +5,7 @@ ignore-regex = "\\\\[fnrstv]"
 # ignore-words-list = ''
 
 [tool.pytest.ini_options]
-addopts = [
-    "--strict-config",
-    "--strict-markers",
-]
+addopts = ["--strict-config", "--strict-markers"]
```

Comment:
(Seems unrelated and status quo was arguably better for future changes with regard to future diffs. Okay to stay but recommending to drop.)

Reply:
It's from my autoformat code editor. Revertes.

Reply:
Thank you!